configure.ac

```diff
@@ -0,0 +1,23 @@
+AC_PREREQ([2.69])
+AC_INIT([iBASIC], [0.1], [ian@mnementh.co.uk], [ibasic], [https://github.com/spyro2/ibasic])
+AC_CONFIG_SRCDIR([ast.c])
+AC_CONFIG_HEADERS([config.h])
```

Comment:
I didn't add any use of it, but there's now a config.h file that might have useful stuff (including things like package name, version, email, URL, etc, from line 2).